authzed/api/v1/md.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+package authzed.api.v1;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = "github.com/authzed/authzed-go/proto/authzed/api/v1";
+option java_package = "com.authzed.api.v1";
+
+extend google.protobuf.EnumValueOptions {
+  optional string string_name = 90001;
+}
+
+// RequestMetadataKey contains the string values used as keys in grpc requests
+// Metadata headers.
+enum RequestMetadataKey {
+  REQUEST_METADATA_KEY_UNSPECIFIED = 0;
+
+  // Overlap is the optional key used to specify which requests shouldoverlap in
+  // CockroachDB transactions.
+  REQUEST_METADATA_KEY_SPICEDB_OVERLAP = 1 [
+    (string_name) = "io.spicedb.reqmeta.overlap"
+  ];
+}
+
+// ResponseMetadataKey contains the string values used as keys in grpc response
+// Metadata headers.
+enum ResponseMetadataKey {
+  RESPONSE_METADATA_KEY_UNSPECIFIED = 0;
+
+  // Request ID is a unique value given to each request.
+  RESPONSE_METADATA_KEY_SPICEDB_REQUEST_ID = 1 [
+    (string_name) = "io.spicedb.respmeta.requestid"
+  ];
+
+  // Server version returns the version of SpiceDB that is serving the request.
+  RESPONSE_METADATA_KEY_SERVER_VERSION = 2 [
+    (string_name) = "io.spicedb.debug.version"
+  ];
+
+  // Dispatched operations count is a count of how many subproblems were
+  // dispatched to another SpiceDB node.
+  RESPONSE_METADATA_KEY_DISPATCHED_OPERATIONS_COUNT = 3 [
+    (string_name) = "io.spicedb.respmeta.dispatchedoperationscount"
+  ];
+
+  // Cached operations count is a count of how many subproblems were
+  // not dispatched to another SpiceDB node because the result was cached.
+  RESPONSE_METADATA_KEY_CACHED_OPERATIONS_COUNT = 4 [
+    (string_name) = "io.spicedb.respmeta.cachedoperationscount"
+  ];
+
+  // Debug info includes additional details about how the request was resolved.
+  RESPONSE_METADATA_KEY_DEBUG_INFO = 5 [
```

Comment:
I think we're missing some request keys, since these are only filled in if requested